service-front/app/test/CommonTest/Middleware/Authentication/AuthenticationMiddlewareTest.php

```diff
@@ -18,15 +20,27 @@ class AuthenticationMiddlewareTest extends TestCase
     /** @test */
     public function it_pipes_a_request_through_all_authentication_middlewares(): void
     {
-        $pipe                               = $this->createMock(MiddlewarePipeInterface::class);
+        $pipe = $this->createMock(MiddlewarePipeInterface::class);
         $credentialAuthenticationMiddleware = $this->createMock(CredentialAuthenticationMiddleware::class);
-        $forcedPasswordResetMiddleware      = $this->createMock(ForcedPasswordResetMiddleware::class);
+        $forcedPasswordResetMiddleware = $this->createMock(ForcedPasswordResetMiddleware::class);
 
         $request = $this->createMock(ServerRequestInterface::class);
         $handler = $this->createMock(RequestHandlerInterface::class);
-
         $response = $this->createMock(ResponseInterface::class);
 
+        $container = $this->createMock(ContainerInterface::class);
+
+        $container->method('get')->willReturnMap([
+                                                     [MiddlewarePipeInterface::class, $pipe],
+                                                     [CredentialAuthenticationMiddleware::class, $credentialAuthenticationMiddleware],
+                                                     [ForcedPasswordResetMiddleware::class, $forcedPasswordResetMiddleware],
+                                                     ['config', [
+                                                         'feature_flags' => [
+                                                             'allow_gov_one_login' => false
+                                                         ],
+                                                     ]],
+                                                 ]);
```

Comment:
```suggestion
        $container->method('get')->willReturnMap(
            [
                 [MiddlewarePipeInterface::class, $pipe],
                 [CredentialAuthenticationMiddleware::class, $credentialAuthenticationMiddleware],
                 [ForcedPasswordResetMiddleware::class, $forcedPasswordResetMiddleware],
                 ['config', [
                     'feature_flags' => [
                         'allow_gov_one_login' => false
                     ],
                 ]],
             ]
         );
```

PHPStorm tends to screw up the indentation when you start an argument list with an array. When that happens you have to force it to a new line like this.